calcom/cal.com
# fix: multiselect required check

## What does this PR do?

- Fixes #16599
- Fixes CAL-4317

## Mandatory Tasks (DO NOT REMOVE)

- [ y] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ y] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ y] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?



- I tested the expected functionality. I deleted the options from multi-select and added them again, the required check is working as expected.

## Checklist
